src/abi.md

```diff
@@ -1,21 +1,246 @@
 # Application Binary Interface (ABI)
 
+r[abi]
+
 This section documents features that affect the ABI of the compiled output of
 a crate.
 
 See *[extern functions]* for information on specifying the ABI for exporting
-functions. See *[external blocks]* for information on specifying the ABI for
+functions. See *[`extern` block]s* for information on specifying the ABI for
 linking external libraries.
 
+## ABI compatibility
+
+r[abi.compatibility]
```

Comment:
There's no paragraph after this marker... is this the normal syntax?

Reply:
Yes, it's an identifier for the whole section.